bindings/resolver.go

```diff
@@ -0,0 +1,253 @@
+package bindings
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// SecretReader represents the read-only content of a secret value.
```

Comment:
Technically this isn't specific to secrets/bindings. Should we move this type out to a more general location in `packit` for use in other places? (renaming to something like `ByteReader` or similar)

Reply:
Sounds good 👍